app/forms/disk-attach.tsx

```diff
@@ -28,6 +29,7 @@ type AttachDiskProps = {
 export function AttachDiskSideModalForm({
   onSubmit,
   onDismiss,
+  disksToFilter,
```

Comment:
I would give this a default value of `[]` so you don't have to do the `?` on 44 and rename it to `disksToExclude` to avoid the ambiguity of filter in vs. filter out. Maybe even go as far as `diskNamesToExclude` as a hint to future callers.